Sources/_AtomicsShims/include/_AtomicsShims.h

```diff
@@ -360,7 +360,14 @@ SWIFTATOMIC_DEFINE_TYPE(COMPLEX, DoubleWord, _sa_dword, uint64_t)
 #error "Unsupported intptr_t bit width"
 #endif // __INTPTR_WIDTH
 
-extern void _sa_retain_n(void *object, uint32_t n);
-extern void _sa_release_n(void *object, uint32_t n);
+static inline void _sa_retain_n(void *object, uint32_t n) {
```

Comment:
You might crib `#define HEADER_SHIM static inline __attribute__((__always_inline__))` from Swift Numerics for this purpose.

Reply:
Good point! In fact I already have it as `SWIFTATOMIC_INLINE`, I just forgot to use it. 🙈